<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicConnectOptionWidget</class>
 <widget class="QWidget" name="BasicConnectOptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="shortCutsGroupBox">
     <property name="title">
      <string>Shortcuts</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>The keyboard shortcuts can be activated in the flight simulator itself, to trigger the corresponding Sky Dolly functionality.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Record:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QKeySequenceEdit" name="recordSequenceEdit">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Replay:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QKeySequenceEdit" name="replaySequenceEdit">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Pause:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QKeySequenceEdit" name="pauseSequenceEdit">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Stop:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QKeySequenceEdit" name="stopSequenceEdit">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Backward:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QKeySequenceEdit" name="backwardSequenceEdit">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Forward:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QKeySequenceEdit" name="forwardSequenceEdit">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Begin:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QKeySequenceEdit" name="beginSequenceEdit">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>End:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QKeySequenceEdit" name="endSequenceEdit">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="extendedOptionGroupBox">
     <property name="title">
      <string>Extended</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
